8/sam-phippen-captions.srt

```diff
@@ -2507,12 +2511,12 @@ So I'm actually just going to write the method.
 
 544
 00:49:23,960 --> 00:49:27,952
-CB: Oh of course, inaud to make it pass.
+CB: Oh of course, fake it to make it pass.
```

Comment:
I've revised this: https://github.com/fromPeerToPeer/captions/commit/76377ccb74947b46cf7ec389af90a4574d3e1537